### PR TITLE
new changes

### PR DESCRIPTION
**Pull request by toppsdown**
this is the first branch merge

@maxamilion  please review!

### CODE REVIEW

File: app/controllers/basic_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class BasicController < ApplicationController
+  def bunnies
+    render 'basic_view'
```
Comment (toppsdown) [inline app/controllers/basic_controller.rb:3]: I like bunnies

File: app/views/basic/basic_view.html [hunk 2]
```diff
@@ -0,0 +1 @@
+hello world
```
Comment (toppsdown) [inline app/views/basic/basic_view.html:1]: @maxamilion

**Comment by maxamilion** [comment]
+1

File: app/views/basic/basic_view.html (hunk 2 above)
Comment (maxamilion) [inline app/views/basic/basic_view.html:1]: You are missing an exclamation mark.

**Pull request merged**